doc/users/next_whats_new/imshow_extent_units.rst

```diff
@@ -0,0 +1,26 @@
+The *extent* parameter of imshow and set_extent functions can now be expressed with units
+-----------------------------------------------------------------------------------------
+The *extent* parameter of `~.axes.Axes.imshow` and  `~.AxesImage.set_extent`
+can now be expressed with units.
+
+.. plot::
+    :include-source: true
+
+    import matplotlib.pyplot as plt
+    import numpy as np
+    from matplotlib.dates import HourLocator, DateFormatter
+    from matplotlib.ticker import AutoMinorLocator
+
+
+    fig, ax = plt.subplots()
+    dates = np.arange("2020-01-01","2020-01-13", dtype='datetime64[h]')
```

Comment:
I'm confused why you need `arange` here when you're only using `dates[0]` and `dates[10]`.

Reply:
fixed it